js/load.js

```diff
@@ -198,6 +201,9 @@ VueRegistry.registerComponent('field-pub-id', FieldPubId);
 // Register ListPanel
 VueRegistry.registerComponent('PkpListPanel', ListPanel);
 
+// Register PublicationSectionJats
+VueRegistry.registerComponent('PkpPublicationJats', PublicationSectionJats);
+
```

Comment:
Lets not register this component here. Here are only general components, so its possible to use them from plugins. Its perfectly sufficients that its 'components' field in workflow.vue

Reply:
I think I found it - I can put `publication-section-jats` because I have named the component `PublicationSectionJats` in the `WorkflowPage.vue`

Fixed.